source/shared/cpp/ObjectModel/Column.cpp

```diff
@@ -112,4 +112,14 @@ std::shared_ptr<Column> Column::Deserialize(const Json::Value& value)
 std::shared_ptr<Column> Column::DeserializeFromString(const std::string& jsonString)
 {
     return Column::Deserialize(ParseUtil::GetJsonValueFromString(jsonString));
+}
+
+std::shared_ptr<BaseActionElement> Column::GetSelectAction() const
```

Comment:
Ok.  Thought you forgot since it wasn't mentioned in the description. :)

---
In reply to: [124076652](https://github.com/Microsoft/AdaptiveCards/pull/441#discussion_r124076652) [](ancestors = 124076652)